src/providers/Profile.js

```diff
@@ -60,6 +70,7 @@ function ProfileProvider({ children }) {
 
   useEffect(() => {
     setProfile(null)
+    setLoadingProfile(true)
```

Comment:
Should we move this after the if statement below?

Or we could actually move it to `fetchAccountProfile`